helpers.lua

```diff
@@ -244,9 +245,29 @@ function helpers.sysctl_async(path_table, parse)
 
     spawn.with_line_callback("sysctl " .. path, {
         stdout = function(line)
-            if not string.find(line, "sysctl: unknown oid") then
-                local key, value = string.match(line, "(.+): (.+)")
-                ret[key] = value
+	    local separators = {
+		freebsd = ": ",
+		linux = " = ",
+		openbsd = "="
+	    }
+            local pattern = ("(.+)%s(.+)"):format(separators[helpers.getos()])
```

Comment:
In our `helpers.lua` there are multiple locations where we call `getos()`. Maybe we could save the output once at startup in a global variable and remove this overhead? (`getos()` is not async) Though, I don't think, it's much overhead ...

Reply:
Actually the output is saved as `kernel_name` so `io.popen` should be called at most once with the module is initialized.  Deprecate the `getos` completely may reduce a bit of overhead (one call and one conditional) but I agree that it's negligible.